scripts/coverage-report.js

```diff
@@ -1008,7 +1010,9 @@ let IndexOfExampleGraphics = indexOfExamples.reduce(function (set, example) {
   let count = example.svgHTML;
   count += example.svgCSS;
   count += example.svgJS;
-  count += example.forcedColorAdjust;
+  //  count += example.forcedColorAdjust;
```

Comment:
Same question as above since it should no longer be used.

Reply:
Yes, I can take it out